script/travis-script-linux

```diff
@@ -1,4 +1,3 @@
 #!/usr/bin/env sh
 
-. ~/.swiftenv/init
```

Comment:
Travis jobs keep failing because of this. Let's revert this as well and run `rake test:swiftpm` directly in Dockerfile (as stated in https://github.com/Quick/Quick/pull/767/files#r249814869).